subprojects/packagefiles/taskflow/unittests/meson.build

```
@@ -0,0 +1,48 @@
+doctest_dep = dependency('doctest', required: false)
+if doctest_dep.found() and get_option('tests')
```

```suggestion
if get_option('tests')
doctest_dep = dependency('doctest', required: true)
```

If the tests are disabled, do not look for doctest at all. If the tests are enabled, then doctest *must* be found.